test/image/mocks/gl3d_errorbars_zy.json

```diff
@@ -0,0 +1,53 @@
+{
+    "data": [
+        {
+            "type": "scatter3d",
+            "x": [
+                1,
+                2,
+                3
+            ],
+            "y": [
+                1,
+                2,
+                3
+            ],
+            "z": [
+                200,
+                100,
+                200
+            ],
+            "error_z": {
+                "type": "percent",
+                "value": 20,
+                "color": "red",
+                "width": 0.5
```

Comment:
This is sposed to add caps, right? Want to make it bigger so we can see them and verify that they end up looking similar to caps that size on 2D svg error bars?